### PR TITLE
feat(wallet): Portfolio Customization Settings

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 
`Portfolio Customization` settings in the `Wallet More Popup`
- Moves `Hide balances` into the `Wallet More Popup`
- Moves `Show graph` into the `Wallet More Popup`

Resolves <https://github.com/brave/brave-browser/issues/29868>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
1. Go to the `Portfolio` page and open the `More Menu` in the header
2. Click on the `Show graph` option and test that the `Price graph` hides or un-hides
3. Refresh the page, the selection should be persisted.
4. Click on the `Hide balances` option and test that the `Balances` hide or un-hides
5. Refresh the page, the selection should be persisted.
6. Open the `Panel` and click on the `More Menu` in the header
7. You should not see any `Portfolio Customization` settings

https://user-images.githubusercontent.com/40611140/233472791-cb8b653c-7950-48d0-9885-98e602521441.mov

### CODE REVIEW

File: components/brave_wallet_ui/common/slices/wallet.slice.ts [hunk 1]
```diff
@@ -128,7 +128,19 @@ const defaultState: WalletState = {
...
+    .localStorage
+    .getItem(
+      LOCAL_STORAGE_KEYS
+        .IS_PORTFOLIO_OVERVIEW_GRAPH_HIDDEN
+    ) === 'true',
+  hidePortfolioBalances: window
+    .localStorage
+    .getItem(
+      LOCAL_STORAGE_KEYS
+        .ARE_PORTFOLIO_BALANCES_HIDDEN
```
[the remark this answers is not in the document]
Comment (onyb) [reply components/brave_wallet_ui/common/slices/wallet.slice.ts:142]: Assuming that these localStorage keys have not made it to Release yet.

**Review by onyb — APPROVED** [review]
Left some minor comments, but looks great otherwise!

**Review by josheleonard — COMMENTED** [review]
maybe rename the new variable name from `ARE_PORTFOLIO_BALANCES_HIDDEN`  to `HIDE_PORTFOLIO_BALANCES` ?

File: components/brave_wallet_ui/common/slices/wallet.slice.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Douglashdaniel) [reply components/brave_wallet_ui/common/slices/wallet.slice.ts]: Updated the new variable to `HIDE_PORTFOLIO_BALANCES`!

**Review by darkdh — APPROVED** [review]
l10n strings ++

**Review by mkarolin — APPROVED** [review]
GRDP ++

**Pull request merged**